README.md

```diff
@@ -30,8 +30,9 @@ Clone the repo using git clone.
 > Start the node app.
```

Comment:
@ShrijanaCodes, let's make sure to add clear instructions to the readme. Step one clone.... step 2... run node index.js.... etc.. and what does the app do?

Reply:
Sounds good, Tim! On it.

Reply:
@timslaglezoom I have updated the README, Tim.